src/objc_func_selector.c

```diff
@@ -0,0 +1,65 @@
+
+/* Compiler implementation of the D programming language
+ * Copyright (c) 2015 by Digital Mars
+ * All Rights Reserved
+ * written by Jacob Carlborg
+ * http://www.digitalmars.com
+ * Distributed under the Boost Software License, Version 1.0.
+ * http://www.boost.org/LICENSE_1_0.txt
+ * https://github.com/D-Programming-Language/dmd/blob/master/src/objc_func_selector.c
+ */
+
+#include "aggregate.h"
+#include "attrib.h"
+#include "declaration.h"
+#include "expression.h"
+#include "id.h"
+#include "objc.h"
+#include "scope.h"
+
+void objc_FuncDeclaration_semantic_setSelector(FuncDeclaration *self, Scope *sc)
```

Comment:
You should add a stub for `objc_FuncDeclaration_semantic_setSelector(..)` in objc_stubs.h, because that way you can avoid including this file if D_OBJC != 1 in the posix.mak file.